### ISSUE TITLE
Feature request: Flashlight intensity. 

### ISSUE THREAD

**Issue by Hamada880** [TheGameCreators/GameGuruRepo#2458]
Asked to post this as requested feature. The ability to change the flashlight intensity away from changing the whole scene mode or effects. it will be awesome if it works like lights and be able to change colors, intensity or to flicker mode.

**Comment by relishrompgames** [comment]
Yes, the flashlight is too dim. Here are good examples of a nice flashlight. Thanks! Love GGMax!!!
https://forum.game-guru.com/attachment/29937
https://forum.game-guru.com/attachment/29938
